### PR TITLE
feat(rust): establish a secure channel to project's authority when waiting for project to be ready

### PR DESCRIPTION
**Pull request by adrianbenavides**

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/project/util.rs [hunk 1]
```diff
@@ -241,8 +243,49 @@ pub async fn check_project_readiness<'a>(
...
+            Ok(sc_addr) => {
+                // Try to delete secure channel, ignore result.
+                let _ = delete_secure_channel(ctx, opts, api_node, tcp, &sc_addr).await;
+            }
+            Err(_) => {
+                loop {
+                    print!(".");
+                    std::io::stdout().flush()?;
+                    tokio::time::sleep(std::time::Duration::from_secs(2)).await;
+                    if let Ok(sc_addr) = create_secure_channel_to_authority(
```
Comment (polvorin) [inline implementations/rust/ockam/ockam_command/src/project/util.rs]: can we simplify this by removing the first special case outside of the loop?   if the intention is to not print "." nor wait the fist time,  can put that at the end of the loop block.

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/project/util.rs]: done!

**Comment by adrianbenavides** [comment]
@mergifyio rebase

**Comment by adrianbenavides** [comment]
@polvorin I'm not sure if this is still needed, but I guess it wouldn't hurt to have this extra check. Let me know what you think so we can move forward with it or close it.

**Comment by etorreborre** [comment]
@adrianbenavides @polvorin is this still needed? Would that help with the timeouts that were recently discussed?

**Comment by adrianbenavides** [comment]
@CAOakleyII is working on an updated version of this, so I'm closing this PR.

**Pull request closed without merging**